dist/licenses/LICENSE-mysql.txt

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
@mrproliu Please be clear about this is dependency. `LICENSE-mysql` seems very strange, it looks like mysql server license, which is GPL, but at here, it is MPL v2 only.

Recommend to use `LICENSE-gosqldriver-mysql`